plugins/trace_api_plugin/test/test_data_handlers.cpp

```diff
@@ -15,10 +15,10 @@ BOOST_AUTO_TEST_SUITE(abi_data_handler_tests)
       auto action = action_trace_v0 {
          0, "alice"_n, "alice"_n, "foo"_n, {}, {}
       };
-      abi_data_handler handler;
+      abi_data_handler handler(exception_handler{});
 
       auto expected = fc::variant();
-      auto actual = handler.process_data(action);
+      auto actual = handler.process_data(action, [](){});
```

Comment:
I didn't catch this in the last review (for 2.0.x) but is there a reason why these are no-op functions instead of `{}` or `nullptr`?  The `optional_delegate` should have a default constructor inherited from `std::function` which will prevent the call right?

Reply:
No, just habit from working with `std::function`